<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>photoShop</class>
 <widget class="QDialog" name="photoShop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图像处理</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pictab">
      <attribute name="title">
       <string>图片</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,7">
       <item>
        <widget class="QDockWidget" name="workbox">
         <widget class="QWidget" name="dockWidgetContents">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="ChPicBtn">
             <property name="text">
              <string>选择图片</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="YuanTuBtn">
             <property name="text">
              <string>显示原图</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="GreyBtn">
             <property name="text">
              <string>灰度化</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="TXTBtn">
             <property name="text">
              <string>3×3均值滤波</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="SideBtn">
             <property name="text">
              <string>边缘检测</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>1000</width>
                 <height>1000</height>
                </size>
               </property>
               <property name="text">
                <string>二值化调节</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>10000</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>100</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QSlider" name="TwoSideSlider">
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="Gamalabel">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>1000</width>
                 <height>1000</height>
                </size>
               </property>
               <property name="text">
                <string>伽马变化</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="GamalineEdit">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>10000</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>2.5</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QSlider" name="GamaSlider">
             <property name="maximum">
              <number>250</number>
             </property>
             <property name="value">
              <number>25</number>
             </property>
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0" colspan="4">
          <widget class="QLabel" name="piclabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="BeforeLabel">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="BeforeBtn">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>上一张</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="AfterBtn">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>下一张</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="AfterLabel">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="vidtab">
      <attribute name="title">
       <string>视频</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
